.github/workflows/ci.yml

```
@@ -0,0 +1,137 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  clang-format:
+    name: 'clang-format'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run clang-format
+        run: |
+          find CONFIG LEGO1 ISLE miniwin -iname '*.h' -o -iname '*.cpp' | xargs \
+          pipx run "clang-format>=17,<18" \
+            --style=file \
+            -i
+      - name: Check diff
+        run: |
+          git diff --exit-code
+
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: ${{ matrix.shell || 'sh' }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { name: 'Linux',          os: 'ubuntu-latest',  dx5: false, build-type: 'Debug', linux: true, werror: true,   clang-tify: true }
+          - { name: 'MSVC (x86)',     os: 'windows-latest', dx5: true,  build-type: 'Debug', msvc: true,  werror: false,  clang-tify: false,  vc-arch: 'amd64' }
+          - { name: 'MSVC (x64)',     os: 'windows-latest', dx5: true,  build-type: 'Debug', msvc: true,  werror: false,  clang-tify: false,  vc-arch: 'amd64_x86' }
```

Isn't this pointless since there isn't a x64 d3drm.dll?
```suggestion
          - { name: 'MSVC (x64)',     os: 'windows-latest', dx5: false,  build-type: 'Debug', msvc: true,  werror: false,  clang-tify: false,  vc-arch: 'amd64_x86' }
```

---

Thanks for spotting the oversight!